content/pages/how_you_can_use_python.md

```diff
@@ -0,0 +1,29 @@
+Title: {{ How You Can Use Python }}
+Content Date: 2018-10-23 22:27
+
+Tags: python, learning, programming, projects, ideas
+
+Authors: Cleveland PyLadies, Daniel Medina
+
+Summary:  A brief summary of how you can use and apply Python to solve real-world problems.
+
+Python is an extremely versatile programming language known for its beginner friendly syntax.  It is used by NASA, Google, Dropbox, Quora, and many others.
+
+Python is primarily used for scripting, the automation of short repetitive tasks.
+
+Here are some of the ways that Python is used across multiple areas:
+
+  Science with [SciPy](https://www.datacamp.com/community/tutorials/python-scipy-tutorial), [NumPy](https://www.machinelearningplus.com/python/numpy-tutorial-part1-array-python-examples/)
+
+  Web Development with [Django](https://simpleisbetterthancomplex.com/series/2017/09/04/a-complete-beginners-guide-to-django-part-1.html) and [Flask](https://www.tutorialspoint.com/flask/)
+
+  Machine Learning with [Scikit-learn](https://medium.com/@mwitiderrick/introduction-to-supervised-learning-with-python-scikit-learn-tutorial-966457dd33b9)
+
+  Deep Learning with [TensorFlow](https://www.tensorflow.org/tutorials/)
+
+  Data Visualization using [Bokeh](https://www.fullstackpython.com/blog/responsive-bar-charts-bokeh-flask-python-3.html) and [Seaborn](https://elitedatascience.com/python-seaborn-tutorial)
+
+
+No matter which area interests you, pick a problem and break it down into manageable pieces.  Approach it step-by-step.  See what you can find online (Google, StackOverflow, YouTube, documentation) when you get stuck, and be sure to ask for help if you've hit an impasse.
```

Comment:
Plug us! haha "See what you can find online...when you get stuck, [come to a meetup](https://www.meetup.com/cle-pyladies), and be sure to ask for help (Slack [here](https://slackin.pyladies.com/) or [here](https://cleveland-tech.slack.com/) is a great place to get feedback on your code) if you've hit an impasse."